src/ethereum_test_tools/spec/eof/eof_test.py

```diff
@@ -48,6 +84,51 @@ def make_eof_test_fixture(
                 }
             }
         )
+        eof_parse = EOFParse()
+        for _, vector in fixture.vectors.items():
+            expectedResult = vector.results.get(str(fork))
+            result = eof_parse.run(input=str(vector.code))
+            self.verify_result(result, expectedResult, vector.code)
+
+        return fixture
+
+    def verify_result(self, result: CompletedProcess, expectedResult: Result, code: Bytes):
+        parser = EvmoneExceptionParser()
+        res_error = result.stdout.replace("\n", "")
```

Comment:
currently eof tool does not return an error code, so we analyze the stdout

Reply:
would be nice to have error codes!